conform-pr/main.go

```diff
@@ -30,7 +56,31 @@ func checkTitle(action *githubactions.Action) error {
 	}
 
 	if strings.HasSuffix(title, ".") {
-		return fmt.Errorf("checkTitle: PR title ends with dot")
+		return fmt.Errorf("checkTitle: PR title must not end with dot, but it does")
+	}
+
+	return nil
+}
+
+// checkBody checks if PR's body (description) ends with a punctuation mark.
+func checkBody(action *githubactions.Action) error {
+	body, err := getPRBody(action)
+	if err != nil {
+		return fmt.Errorf("checkBody: %w", err)
+	}
+
+	// It is allowed to have empty body.
+	if len(body) == 0 {
+		return nil
+	}
+
+	match, err := regexp.MatchString(`.*([.!?])$`, body)
```

Comment:
It looks inappropriate to me when a PR body has a question mark `?`: seems like one is in doubt? 
When it ends with `!`, then it is OK.

Reply:
Well, potentially it can be ended with a question mark. For example, if the last part of the body is called "Open questions" which contains a list of questions. Even worse, the last symbol could be an emoji! 🙀

But I don't have a strong opinion, and probably in the future, we will modify this github action again (e.g. trim empty lines or whitespace completely). 

@AlekSi any thoughts? Should I keep the question mark or should I delete it?

Reply:
I think both doubts and excitement are appropriate in the PR body. Even both at the same time?!